src/main/kotlin/com/openlattice/organizations/OrganizationMetadataEntitySets.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2020. OpenLattice, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * You can contact the owner of the copyright at support@openlattice.com
+ *
+ *
+ */
+
+package com.openlattice.organizations
+
+import java.util.*
+
+/**
+ *
+ * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
+ */
+data class OrganizationMetadataEntitySets(
+        val organization: UUID = UUID(0, 0),
+        val datasets: UUID = UUID(0, 0),
+        val columns: UUID = UUID(0, 0)
```

Comment:
I prefer `properties` over `columns` since the context here is `EntitySets`.

Reply:
Datasets have columns and entity sets have properties. I would say we are emphasizing Research Data Hub so we should use columns.